src/pymor/models/basic.py

```diff
@@ -122,13 +128,43 @@ def __init__(self, operator, rhs, outputs=None, products=None,
         super().__init__(products=products,
                          estimator=estimator, visualizer=visualizer,
                          cache_region=cache_region, name=name)
+        self.linear = operator.linear
```

Comment:
Maybe this should be moved to a different PR? There should also be a check that `rhs` is actually linear.

Reply:
This was used for the previous version of `StationaryModel.__str__`, which also says if the problem is linear or not.

Reply:
What I meant to ask is, do we need the `linear` attribute?